Frontend/src/Components/SystemPower/SystemPower.js

```diff
@@ -0,0 +1,92 @@
+import {Flex, useColorMode} from "@chakra-ui/react";
+import RangeRow from "../Shared/RangeRow";
+import CONSTANTS from "../../data-constants.json";
+import getColor from "../Shared/colors";
+
+export default function SystemPower(props) {
+    const { colorMode } = useColorMode();
+    const headerBg = getColor("header", colorMode);
+
+    return (
+        <Flex flex="auto" direction="column" overflowY="scroll">
+            <RangeRow
+                dataTitle="BMS Input Voltage"
+                dataValue={props.data?.bms_input_voltage[0] ?? -1.0}
+                dataConstant={CONSTANTS.bms_input_voltage}
+                DecimalPoint='3'
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Main VBus Curr"
```

Comment:
```suggestion
                dataTitle="Main VBus Current"
```
Let's expand this to say "Current." Looking at the image you submitted, I don't think it will overflow normally.